src/app/submission/sections/container/section-container.component.html

```diff
@@ -12,23 +12,26 @@
                  [destroyOnHide]="false">
     <ngb-panel id="{{ sectionData.id }}">
       <ng-template ngbPanelTitle>
-        <span class="float-left">{{ 'submission.sections.'+sectionData.header | translate }}</span>
+        <span class="float-left section-title" tabindex="0">{{ 'submission.sections.'+sectionData.header | translate }}</span>
         <div class="d-inline-block float-right">
           <i *ngIf="!(sectionRef.isValid() | async) && !(sectionRef.hasErrors())" class="fas fa-exclamation-circle text-warning mr-3"
-             aria-hidden="true"></i>
+             aria-hidden="true" title="{{'submission.sections.status.warnings.title' | translate}}"></i>
           <i *ngIf="(sectionRef.hasErrors())" class="fas fa-exclamation-circle text-danger mr-3"
-             aria-hidden="true"></i>
+             aria-hidden="true" title="{{'submission.sections.status.errors.title' | translate}}"></i>
           <i *ngIf="(sectionRef.isValid() | async) && !(sectionRef.hasErrors())" class="fas fa-check-circle text-success mr-3"
-             aria-hidden="true"></i>
-          <a class="close" aria-label="Close">
+             aria-hidden="true" title="{{'submission.sections.status.valid.title' | translate}}"></i>
+          <a class="close"
+             tabindex="0"
+             [attr.aria-label]="(sectionRef.isOpen() ? 'submission.sections.toggle.close' : 'submission.sections.toggle.open') | translate"
```

Comment:
For some reason Axe scanner is complaining about this `<a>` tag including an `aria-label`...it says that this `aria-label` is not allowed based on the "element's role".  I wonder if there's a way to just turn this into a `<button>`?

Reply:
fixed by adding a role button to <a> tag